### ISSUE TITLE
[WCAG 2.2 Draft Feedback] Missing href for Understanding Success Criterion 3.3.7: Accessible Authentication

### ISSUE THREAD

**Issue by JohnRiv** [w3c/wcag#1509]
The link to "Success Criterion 1.3.5 Input Purpose" in [Understanding Success Criterion 3.3.7: Accessible Authentication](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication.html#intent) currently is a broken link (`href="#"`).

In reviewing the current WCAG 2.2 Draft, it seems the href value should be `https://www.w3.org/TR/WCAG22/#identify-input-purpose` and the text for that link should be updated to reflect the section name: "Success Criterion 1.3.5 **Identify** Input Purpose"

**Comment by alastc** [comment]
Thanks @JohnRiv, that's already been updated in the [latest version](https://w3c.github.io/wcag/understanding/accessible-authentication.html), so I'll close this one.
